waterlog/src/Components/Tank.js

```diff
@@ -1,50 +1,15 @@
 import React from "react";
-import "../Stylesheets/_tank.scss";
-const images = {
-  tank_yellow: "images/tank-yellow.png",
-  tank_orange: "images/tank-orange.png",
-  tank_green: "images/tank-green.png",
-  tank_red:"images/tank-red.png"
-};
+import { getTankImage } from './../utils';
 
 const Tank = props => {
-  const getTankImage = (percent) => {
-    if(percent===0){
-      return (
-        <div>
-          <img src={images.tank_red} alt="tank" />
-          <p id="homepage_tank_percentage_lightimage">{percent}%</p>
-        </div>
-      );
-    }
-    else if (percent >= 1 && percent <= 40) {
-      return (
-        <div>
-          <img src={images.tank_yellow} alt="tank" />
-          <p id="homepage_tank_percentage_lightimage">{percent}%</p>
-        </div>
-      );
-    } else if (percent >= 41 && percent <= 79) {
-      return (
-        <div>
-          <img src={images.tank_orange} alt="tank" />
-          <p id="homepage_tank_percentage_lightimage">{percent}%</p>
-        </div>
-      );
-    } else if (percent >= 80 && percent <= 100) {
-      return (
-        <div>
-          <img src={images.tank_green} alt="tank" />
-          <p id="homepage_tank_percentage">{percent}%</p>
-        </div>
-      );
-    }
-  };
-
+	const { id, percentageLevel } = props.tank;
   return (
-    <div className="tank">
-      <p>Tank {props.tank.id}</p>
-      {getTankImage(props.tank.percentageLevel)}
+		<div className="tank" onClick={() => (window.location = `/alert/tank/${id}`)}>
+			<p>Tank {props.tank.id}</p>
+			<img src={getTankImage(percentageLevel)}
+				className={(percentageLevel === 0 || percentageLevel > 80) ? 'exlamation': ''}
```

Comment:
spelling mistake `exlamation` -> `exclamation

Reply:
i was wondering why it was not working